android/modules/ui/src/java/ti/modules/titanium/ui/widget/webview/TiUIWebView.java

```diff
@@ -647,6 +647,22 @@ public void setUrl(String url)
 		}
 	}
 
+	public void injectCode()
+	{
+		StringBuilder code = new StringBuilder();
+		StringBuilder tiCode = TiWebViewBinding.readResourceFile("binding.min.js");
+		String outCode = tiCode.toString().replaceAll("\"", "\\\\\"")
+			.replace("\n", "").replace("\r", "");
+		code.append("(function addBinding(){");
+		code.append("var s=document.createElement(\"script\");");
+		code.append("s.setAttribute(\"type\",\"text/javascript\");");
+		code.append("s.innerHTML=\"" + outCode + "\";");
+		code.append("document.getElementsByTagName(\"body\")[0].appendChild(s);");
+		code.append("})()");
+		Log.i("---", code.toString());
+		getJSValue(code.toString());
+	}
```

Comment:
Since the heavy lifting is all done in the `TiWebViewBinding`, can we move this logic there and make this function a passthrough?

```suggestion
	public void injectEventBindings()
	{
		client.getBinding().injectEventBindings();
	}
```